workflows/argo-cd/Workflow_Install ArgoCD.yaml

```diff
@@ -243,14 +240,18 @@ spec:
     key: argoCDSecret
     requiresValue: false
     showOnReleaseStart: false
-  scriptUsername: admin
+  - type: xlrelease.StringVariable
+    key: runAsUsername
+    showOnReleaseStart: false
+  - type: xlrelease.PasswordStringVariable
+    key: runAsUserPassword
+    showOnReleaseStart: false
```

Comment:
Add label and description for these variables in all templates which are modified. Description will be populated on user input task field, so user knows what he/she is suppose to set.

Reply:
Follow same standard across all variable naming. There are different keys like `runAsUsername`, `run-as-user` etc.

Reply:
Variables have been renamed to follow a standard. Descriptions have been added for all variables.

Reply:
Renamed variables to follow a standard and added descriptions.